### ISSUE TITLE
Spelling fixes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8630]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8630|
|Type|enhancement|
|Created|2016-07-19 20:48:24Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8630 8630
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468961304537400 1468961304537400
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/358

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 14fd13a32e741a5bc443dbb662dc405306609319
```
#!CommitTicketReference repository="" revision="14fd13a32e741a5bc443dbb662dc405306609319"
Merge pull request #364 from twisted/8630-scop-spelling

Author: scop
Reviewer: rodrigc
Fixes: #8630

Spelling fixes
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
